---
title: "tRFdb paper"
author: "Elizabeth Ferrick Kiddie"
date: "3/31/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)

```

##Navigate to the tRNA fragment database (tRFdb):
http://genome.bioch.virginia.edu/trfdb/

##Here is the link to read the paper that describes the tRFdb website:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4383946/

##Here are some questions to consider while reading their paper:

1) What type of tRNA fragments did they characterize? (Note which type of tRNA fragment has nucleotides not found in the tRNA gene)
</br></br>
<details>
  <summary>Click for Answer</summary>
    
    </br>![](/Users/elizabethferrick/Desktop/tRF_processing_fig_adapt_Kumar2015.jpeg)
    </br></br></br>3' tRFs (tRF-3) have CCA nucleotides added after transcription from the tRNA gene, so they will not perfectly match the genome anymore.
    </br></br>They address how they modified their RNA reference sequences in the methods section:
<span style="color: red;">"We extracted mature tRNA genes from the same genome assembly on which the tRNA gene coordinates were built and added ‘CCA’ at the end. In addition to that we also add 50 bases downstream sequences to find the tRFs generated from the tRNA trailer sequences."</span>

</details> 

***
***

2) What length of the 5' and 3' tRNA fragments (tRF-5 and tRF-3) do Kumar *et al.* (2015) find in humans?
</br></br>
<details>
  <summary>Click for Answer</summary>

    From the results section:
<span style="color: red;">"For humans and mice, tRF-5s are mainly 15, 22 and 32 nts, whereas tRF-3s are 18 and 22 bases long"</span>

</details> 

***
***

3) Will tRFdb include tRNA reads that have mismatches? (Note: mismatches in sequencing can happen at RNA bases that have extra modifications added, because the reverse transcriptase incorporates a different nucleotide than it would at an unmodified base)
</br></br>
<details>
  <summary>Click for Answer</summary>

    From the methods section:
<span style="color: red;">"We considered only those alignments where the query sequence (small RNA) was mapped to the database sequence (tRNA) along 100% of its length with 100% identity. To eliminate any false positives, the small RNAs that mapped on to the ‘tRNAdb’ were again searched against the whole genome database using blast search excluding the tRNA loci. Only those small RNAs that mapped exclusively to tRNA loci were included as probable tRFs."</span>

</details> 
</br></br>
<details>
  <summary>Liz's notes on what question 3 means for us</summary>

<style>
div.blue { background-color:#e6f0ff; border-radius: 5px; padding: 20px;}
</style>
<div class = "blue">
This means that if an RNA sequence had even one letter different than the RNA reference sequence (except for the CCA tail which they added to the reference), they threw it out and didn't include it in the database. This is a stringent approach, and for their purposes it makes sense that they decided to be stringent, because they are building a huge database and want to categorize the most abundant fragments simultaneously. But it also means that they might be missing data from tRNA fragments that likely have RNA modifications that can affect what nucleotide gets incorporated during reverse transcription. Our mapping pipeline doesn't throw out tRNA fragments that have some mismatches, so we keep more data, but it complicates the mapping and means I have to look at the data by eye more often...so the data generated by our lab will look a little different than what we get from tRFdb. It's not something right or wrong about the way the data is analyzed, just something we need to keep in mind.</div>

</details> 
</br></br>